version.txt

```diff
@@ -1 +1 @@
-v5.34.0
+v6.0.0
```

Comment:
This is arguable and will depend on the solution, it might be also entirely possible that we manage to create something that is entirely backwards compatible workflow-wise, making this into a `5.35.0`! Needs more tests and info from Veracode related to features we use in some of our workflows **(e.g.: skipping some modules or using a specific Maven profile!)**

Reply:
Yeah, those need to be placed in srcclr.yml file in the respective repository with this approach, can't really be in command execution, but you could theoretically hack it to use the older input and modify the srcclr.yml file in this action before the scan is ran with sed or something? I'm not 100% sure that would work though.

Reply:
I've tested with alfresco-transform-service, on master branch

I've added a line in srcclr.yml of that project
`custom_maven_command: clean install -DskipTests -pl "!test-end-to-end,!test-load"`
and I've used command: 
`srcclr scan --loud --allow-dirty`

Result:
Full Report Details                            https://sca.analysiscenter.veracode.com/teams/QmmUV9P/scans/70631289

Also no mention of testing modules in srcclr execution logs.

--loud option is to print full logs, 
--allow-dirty allows for running scans on non-commited changes - in case we decide to modify this file with action while keeping it backwards compatible. Theoretically we could also push the change, but that might complicate it.

Reply:
I'm ok with doing it the clean way _(by editing the `srcclr.yml` file in reach repository)_ and having this as a breaking change.